src/Tests/SenseNet.ContentRepository.Tests/UserProfileTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.Extensions.Configuration;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SenseNet.Configuration;
+using SenseNet.ContentRepository.Storage;
+using SenseNet.ContentRepository.Storage.Security;
+using SenseNet.Extensions.DependencyInjection;
+using SenseNet.Tests.Core;
+
+namespace SenseNet.ContentRepository.Tests
+{
+    [TestClass]
+    public class UserProfileTests : TestBase
+    {
+        [TestMethod]
+        public void UserProfile_Create()
+        {
+            Test(repoBuilder =>
+            {
+                // switch UserProfilesEnabled config value ON
+                var config = new ConfigurationBuilder()
+                    .AddInMemoryCollection(new Dictionary<string, string>
+                    {
+                        {"sensenet:identityManagement:UserProfilesEnabled", "true"}
+                    })
+                    .Build();
+
+                repoBuilder.UseConfiguration(config);
+            }, () =>
+            {
+                CreateProfileTemplate();
+
+                var parentPath = "/Root/IMS/BuiltIn/Temp-" + Guid.NewGuid();
```

Comment:
Note: every test - which uses the Test method - builds a brand new database so Guid in a name is not required.